CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## [Unreleased]
+
+## [0.1.0] - 2022-01-06
+
+- Fixing typo "require 'observer'"
```

Comment:
please remove

Reply:
So many mistakes, I worked on two devices. Deleted `observer` and `jenncad-fork`, in `Gemfile` and `CHANGELOG.md`